inst/examples/12_counter/app.R

```diff
@@ -0,0 +1,30 @@
+ui <- fluidPage(
+  # ======== Modules ========
+  # mymoduleUI is defined in R/my-module.R
+  mymoduleUI("mymodule1", "Click counter #1"),
+  mymoduleUI("mymodule2", "Click counter #2"),
+  # =========================
+  wellPanel(
+    sliderInput("size", "Data size", min = 5, max = 20, value = 10),
+    div("Lexically sorted sequence:"),
```

Comment:
Instead of hard-coding this, add some metadata which has both options, one of which we keep, depending on choices given to `shinyAppTemplate()`